README.md

```diff
@@ -1,100 +1,275 @@
+We will use Rails to construct a RESTful API. The purpose of this API is to quickly serve information about the store's video collection, customer information, and to update rental status. This repository provides two JSON datafiles to serve as the initial seeds for this system.
+
+This is a pair project. You and your partner should use all the techniques we've learned so far to keep yourselves organized and on track, and ensure that no requirements slip through the cracks.
+
+## Learning Goals
+Upon completing this project, students should be able to:
+
+- Build an ERD and set up ActiveRecord models for a given dataset / use-case
+- Expose database contents through a web API
+- Respond reasonably to bad user data in the context of an API
+- Verify the correctness of an API using controller tests
+
+This is a [stage 2](#) project.
+
+## Success Criteria
+Your project will be evaluated against the following requirements:
+
+- API conformance
+  - The provided test script should pass
+  - Bad data sent to the API should result in helpful errors
+- Test coverage
+  - Models: All relations, validations, and custom model methods should include at least one positive and one negative test case
+  - Controllers: Every API endpoint should include at least one positive and one negative test case
+- Style and Organization
+  - Business logic should be live in models
 
 ## Project Baseline
-- Read the API Requirements below and create a pseudo-code "routes" file that
-  - defines the _endpoints_ your API will need
-  - the _HTTP verbs_ each endpoint will use
-  - and any data that must be provided to the endpoint in order for it to do its work.
-- Create a new Node/Express app to serve as the API.
-- Create a route that responds to `/zomg` that serves a json-encoded "it works!" method.
+- Read the API Requirements below and create a pseudo-code "routes" file that specifies
+  - The _endpoints_ your API will need
+  - The _HTTP verbs_ each endpoint will use
+  - Any data that must be provided to the endpoint in order for it to do its work
+- Create a new Rails app to serve as the API
+- Create a route that responds to `/zomg` that serves a json-encoded "it works!"
 
 ## Wave 1: Database Models, Tables, & Seeds
-- Create an ERD for your database by reading through all of the requirements below.
-- Be sure your ERD includes primary keys for all tables
-- Leverage MassiveJS and create custom npm scripts to...
-  - create a development database
-  - drop a development database
-  - apply a schema to the database
-  - seed a database using the provided json data in `db/seeds`
+- Create an ERD for your database by reading through all of the requirements below
+- Generate Rails models and associations to match your ERD
+- Write a seed script that imports the provided JSON data from `db/seeds` into your database
```

Comment:
Here is my `seeds.rb`

```ruby
JSON.parse(File.read('db/seeds/customers.json')).each do |customer|
  Customer.create!(customer)
end

JSON.parse(File.read('db/seeds/movies.json')).each do |movie|
  Movie.create!(movie)
end
```

I think I would be fine with just giving this to students, since it doesn't have anything to do with the learning goals.